assets/js/blocks/mini-cart/utils/data.ts

```diff
@@ -23,21 +23,40 @@ export const updateTotals = ( totals: [ string, number ] | undefined ) => {
 		'.wc-block-mini-cart__amount'
 	);
 
-	miniCartButtons.forEach( ( miniCartButton ) => {
-		miniCartButton.setAttribute(
-			'aria-label',
-			sprintf(
-				/* translators: %s number of products in cart. */
-				_n(
-					'%1$d item in cart, total price of %2$s',
-					'%1$d items in cart, total price of %2$s',
+	miniCarts.forEach( ( miniCart ) => {
```

Comment:
Same here, following `frontend.ts`:

```suggestion
	miniCartBlocks.forEach( ( miniCartBlock ) => {
	if ( ! ( miniCartBlock instanceof HTMLElement ) ) {
		return;
	}
```